sachin/assignment1.go

```diff
@@ -0,0 +1,337 @@
+package main
```

Comment:
why are there 2 same files? Dont need the first one I guess

Reply:
Deleted first one.